src/content/pages/community-activities.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Community Organisers Activities
+url: /community-activities
+subtitle:
+  Booths, Open spaces, and more activities for organisers
+---
+
+# Community Organisers Activities
+
+The Python community is an essential part of the language, and for many people
+the reason for sticking around and keep meetups, conferences, forums, and many
+more things running, in order to help others.
+
+This year at EuroPython, we have a few activities focused on some of the
+communities around Europe and world, but also initiatives around Python.
+
+
+## Community Organisers Open Space
+
+Communication is the key to avoid repetition and failing on the same mistakes.
+Let's be together and share the experience of our communities and our personal
+challenges, and wins! so more communities can improve.
+
+This year, we are saving a 2-hour slot in the Open Spaces for all the community
+and conference organisers that want to share their experiences, and more
+importantly, to see how the EuroPython Society can help you to achieve your
+goals and solve your current issues.
+
+* Room: TBD
+* Time: TBD
+
+## Community Organisers Lunch
+
+Do you know all the organisers from other European communities and conferences?
+If not, you have a good opportunity to meet more people, that like you, are
+doing their best for keep more Python events and communities running,
+so more people have the chance of learning and improving at Python!
+
+We will be meeting at TBD during TBD for a special lunch, where you will be
```

Comment:
Friday, 18 July, 12:55, Zoom, First floor (entrance near the registration area :))